backend/src/controllers/review.controller.ts

```diff
@@ -63,3 +60,50 @@ export async function editReviewByIdHandler(req, res: Response) {
     });
   }
 }
+
+export async function deleteReviewHandler(req, res) {
+  try {
+    const movieDelete = await prisma.movieRequest.findFirst({
+      where: {
+        id: req.body.id,
+      },
+    });
+
+    if (movieDelete.userId == req.user.id) {
+      res.status(401).json;
```

Comment:
you need to pass params to .json, its a method
ik i cut it off in my example lol